docs/development.md

```diff
@@ -168,3 +170,75 @@ In the vagrant box, find the Hammer repositories at `/home/vagrant/` and the
 configuration at `/home/vagrant/.hammer`. Specifically, to change the Foreman
 instance Hammer points to, update
 `/home/vagrant/.hammer/cli.modules.d/foreman.yml`.
+
+## Capsule Development
+
+To use this functionality, add the following configuration to your boxes.yaml,
+changing the hostnames as needed
+
+### To setup a capsule with an existing development environment
+
+* Add to existing development server's configuration in boxes.yaml
+```
+  ansible:
+    group: 'server'
+```
+* Add capsule with development server's name in server to boxes.yaml:
```

Comment:
How about: Add a capsule development server. Enter the Katello development server as the server.